jasper_load/__manifest__.py

```diff
@@ -29,11 +29,12 @@
 
```

Comment:
index.html pan update karjo.. mention specially k jasper_load is necessary... u know ne.. public shout

Reply:
and also All those Points which we had to reply on Github Issue/Odoo App page.